### PR TITLE
[15.0] [FIX] account_invoice_triple_discount: Fix precision when compute discount

### PR DESCRIPTION
**Pull request by ramiadavid**
Fix the bugs described here #1240

### CODE REVIEW

**Review by Saran440 — APPROVED** [review]

**Comment by ramiadavid** [comment]
@OCA/accounting-maintainers Can someone merge this? Thank you!

**Review by BT-anieto — APPROVED** [review]

**Comment by BT-anieto** [comment]
@OCA/accounting-maintainers Could this PR be merged? Thank you!

**Comment by BT-anieto** [comment]
@dreispt Hello Daniel! Could you also check this PR, please? Thank you very much.

**Comment by BT-anieto** [comment]
Hello @pedrobaeza! This PR has been opened long time ago and although the maintainers have been mentionned, nobody has merged. Could you take a look, please? Thank you very much (again) :)

**Comment by pedrobaeza** [comment]
If you don't review other PRs, don't expect that others review yours. A PSC review is needed for merging, so you can give them reasons to do it contributing a bit more. Don't you think?

**Comment by BT-anieto** [comment]
Thank you for your comment @pedrobaeza I'm on it. If I find any bugs I solve try to solve them. Sometimes there is already a PR for it (like this one), so I review. Sometimes there is no PR, so I create.

File: account_invoice_triple_discount/models/account_move.py [hunk 1]
```diff
@@ -16,6 +16,8 @@ def _recompute_tax_lines(self, **kwargs):
         restored after the original process is done
         """
         old_values_by_line_id = {}
+        digits = self.line_ids._fields["price_unit"]._digits
```
Comment (pedrobaeza) [inline account_invoice_triple_discount/models/account_move.py:23]: Please add a comment indicating why you are doing this for later references without having to do git blame and similar "arcane techniques".

**Comment by pedrobaeza** [comment]
OK, I count with that good faith, and I have made a small comment for being handled and I'll merge this specific one.

**Comment by pedrobaeza** [comment]
Thank you for the change, David. Merging:

/ocabot merge nobump

**Comment by pedrobaeza** [comment]
Agh, my fault, it should be

/ocabot merge patch

Now I abort the bad one

**Comment by ramiadavid** [comment]
@pedrobaeza Is there a problem with this PR?

**Pull request merged**